src/components/hero.jsx

```
@@ -75,67 +75,85 @@
-              >
-                {item.name}
-              </div>
-            ))}
-        </div>}
+            <FaSearch className='text-xl mr-1 mt-1' />
+          </button>
+        </div>
+        { data.length > 0 && <div className='grid items-center opacity-0 animate-fadeIn'>
+          { searchTerm.length > 1 && <h2 className={showDropdown ? 'block mb-5 text-lg font-medium' : 'hidden'}>Search Result : </h2>}
+          { searchTerm.length > 1 && <div className={showDropdown ? 'flex flex-wrap justify-center gap-4 mt-[-20px] mb-5 p-2.5 bg-white border border-gray-300 rounded-2xl max-w-[800px]' : 'hidden'}>
+            <div role="listbox" aria-label="Search results">
```

Biome (1.9.4)

[error] The HTML element with the interactive role "listbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "listbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)